ConfigSpace/conditions.pyx

```diff
@@ -181,6 +183,35 @@ cdef class AbstractCondition(ConditionComponent):
 cdef class EqualsCondition(AbstractCondition):
 
     def __init__(self, child: Hyperparameter, parent: Hyperparameter, value: Union[str, float, int]) -> None:
+        """
+        Adds on the ``child`` hyperparameter the condition, that the ``parent``
+        hyperparameter has to be equal to ``value``.
+
+        Example
+        -------
+
+        >>> import ConfigSpace as CS
+        >>> import ConfigSpace.hyperparameters as CSH
+        >>> cs = CS.ConfigurationSpace()
+        >>> a = CSH.CategoricalHyperparameter('a', choices=[1, 2, 3])
+        >>> b = CSH.UniformFloatHyperparameter('b', lower=1., upper=8., log=False)
+        >>> cs.add_hyperparameters([a, b])
+        # makes 'b' an active hyperparameter if 'a' has the value 1
+        >>> cond = CS.EqualsCondition(b, a, 1)
+        >>> cs.add_condition(cond)
+
+        Parameters
+        ----------
+        child : :ref:`Hyperparameters`
+            This hyperparameter will be sampled in the configspace
+            if the ``equal condition`` is satisfied
+        parent : :ref:`Hyperparameters`
+            The hyperparameter, which has to satisfy the ``equal condition``
```

Comment:
I think it would be good if ``equal condition`` was not formatted as code, but rather in italics or bold face.